i3lock.c

```diff
@@ -628,6 +628,17 @@ static void xcb_prepare_cb(EV_P_ ev_prepare *w, int revents) {
 }
 
 /*
+ * Try closing logind sleep lock fd passed over from xss-lock, in case we're
+ * being run from there.
+ *
+ */
+static void maybe_close_xss_fd() {
```

Comment:
nit: use `static void maybe_close_xss_fd(void) {` for functions which don’t expect parameters.